web-service-consumer/1.6/modules/ROOT/pages/web-service-consumer-config-topics.adoc

```diff
@@ -0,0 +1,92 @@
+= Web Service Consumer Additional Configuration Information - Mule 4
+:page-aliases: connectors::web-service/web-service-consumer-configure.adoc
+
+Set additional configuration for Anypoint Connector for Web Service Consumer (Web Service Consumer Connector), such as custom HTTP Transport and Web Service Security both in Anypoint Studio and XML editor.
+
+== Configure Custom HTTP Transport
+
+By default, the connector uses a simple nonprotected HTTP configuration to dispatch all outgoing SOAP messages. For cases where a more advanced HTTP configuration is required, you can parameterize the Web Service Consumer Connector with an HTTP Connector configuration to dispatch the messages.
+
+In the following example, you declare and reference an HTTP Requester configuration with HTTP Basic authentication in the *Transport Configuration* parameter of the Web Service Consumer Connector configuration, so the connector dispatches each requests using the advanced configuration:
+
+. In Studio, open the *Global Element Properties* window of the Web Service Consumer Connector.
+. On the *Transport* tab, for *Transport Configuration* select `HTTP Transport Configuration`. +
+The *Requester Config* field automatically populates `HTTP_Request_configuration`.
+. Click *OK*
+
+In Studio, configuring Custom HTTP Transport looks as follows:
+
+.Configure Custom HTTP Transport
+image::web-service-consumer-configure-transport.png[Configure Custom HTTP Transport]
+
+In the *Configuration XML* tab, the reference of the HTTP Requester configuration `<http:request-config name="basic-auth">` is under the Web Service Consumer configuration `<wsc:custom-transport-configuration>`, and looks as follows:
+
+[source,xml,linenums]
+----
+<http:request-config name="basic-auth">
+    <http:request-connection host="www.exampleplace.com" port="8182" protocol="HTTP">
+        <http:authentication>
+            <http:basic-authentication username="admin" password="textpassword"/>
+        </http:authentication>
+    </http:request-connection>
+</http:request-config>
+
+<wsc:config name="with-custom-transport">
+    <wsc:connection wsdlLocation="http://www.host.com?WSDL" service="Service" port="Port">
+        <wsc:custom-transport-configuration>
+            <wsc:http-transport-configuration requesterConfig="basic-auth"/>
+        </wsc:custom-transport-configuration>
+    </wsc:connection>
+</wsc:config>
+----
+
+
+
+== Configure Web Service Security
+
+The WS-Security protocol specifies how you can enforce integrity and confidentiality on messages and enables the communication of various security token formats. Its main focus is the use of XML Signature and XML Encryption to provide end-to-end security.
+
+WS-Security describes three main mechanisms:
```

Comment:
A better wording would be "WS-Security describes how to:"

* Sign ...
* Encrypt...
* Attach...